docs/concepts/aws.md

```diff
@@ -0,0 +1,26 @@
+# AWS
+
+AWS is one of the cloud platforms supported by Lokomotive. This document explains various architecture decisions specific to this platform.
+
+## Blocked EC2 instance metadata access
+
+By default, the access to [EC2 instance metadata](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) is blocked for all pods. This is to prevent possible exploitation of IAM roles which might be attached to the cluster nodes.
+
+If your deployment relies on on those roles, you can allow it to access the metadata service by creating a NetworkPolicy, similar to the following example:
```

Comment:
IIUC a NetworkPolicy can override a GNP. Is that correct?

Reply:
Yes, as far as I know.